test/data/test_reference_files/IMG_1361.dng.out

```diff
@@ -139,7 +139,7 @@ Exif.Photo.SceneType                         Undefined   1  1  Directly photogra
 Exif.Photo.ExposureMode                      Short       1  0  Auto
 Exif.Photo.WhiteBalance                      Short       1  0  Auto
 Exif.Photo.FocalLengthIn35mmFilm             Short       1  26  26.0 mm
-Exif.Photo.LensSpecification                 Rational    4  807365/524263 6/1 8/5 12/5  807365/524263 6/1 8/5 12/5
+Exif.Photo.LensSpecification                 Rational    4  807365/524263 6/1 8/5 12/5  1-6mm F1.6-2.4
```

Comment:
Yes, I noticed it as well. Also that the devisor is 524263, where we usually find a 10 or 100. Would be nice to know, if the data were stored in this by the iPhone or if it was changed later.

Reply:
@norbertwg 

>Yes, I noticed it as well. Also that the devisor is 524263, where we usually find a 10 or 100. Would be nice to know, if the data were stored in this by the iPhone or if it was changed later.

I don't have an iPhone but I found some files in https://github.com/thorsted/digicam_corpus/tree/master/Apple . The readme states that they have removed the GPS but I don't know if this has affected any of the other values. I tried a couple of the files and while Apple seem to use more unusual denominators, they didn't match the above example.

`test/data/IMG_1361.dng` was added [here](https://github.com/Exiv2/exiv2/commit/dbf2c1c52dbe6ce439a31f796fd5b629110dfdf3) but I can't account for the unusual value. The ExifTool output of `Exif.Photo.LensSpecification` (called `LensInfo`) is:

```shell
$ exiftool -*lens* IMG_1361.dng
Lens Info                       : 1.539999962-6mm f/1.6-2.4
Lens Make                       : Apple
Lens Model                      : iPhone 12 Pro back triple camera 4.2mm f/1.6
Lens ID                         : iPhone 12 Pro back triple camera 4.2mm f/1.6
```

Reply:
@norbertwg 
> Yes, I noticed it as well. Also that the devisor is 524263, where we usually find a 10 or 100. Would be nice to know, if the data were stored in this by the iPhone or if it was changed later.

Thinking about this again, `807365/524263 = 1.539999962` but we output `1`. We could round the focal length values to 1 decimal place, as this is what Apple use in the file, i.e.,

```shell
Exif.Photo.LensModel                         Ascii      45  iPhone 12 Pro back triple camera 4.2mm f/1.6  iPhone 12 Pro back triple camera 4.2mm f/1.6
```

Reply:
@postscript-dev 
>  We could round the focal length values to 1 decimal place, as this is what Apple use in the file

Also other manufacturers round to 1 decimal place - at least as far as I have seen it. In all other cases than this I have seen, that the current implementation (which follows the example of other rational conversions)
`os << std::setprecision(5) << focalLength1;`
gives this result, because the rational value results in a maximum of 1 decimal places. Here we get 2 decimals, but for me - looking also at the other examples - it is rather sure, that the value was manipulated. So I would not give this example too much importance.